src/ngraph/builder/quantization_util.hpp

```diff
@@ -125,7 +126,37 @@ namespace ngraph
                 // s32 = f32 * std::pow(2, 31)/ max_abs32;
                 // s8 = f32 * std::pow(2, 7)/ max_abs8;
                 // s8 = s32 * std::pow(2, -24) * max_abs32 / max_abs8;
-                return make_constant(type, shape, std::pow(2, -24)) * (max_abs32 / max_abs8);
+
+                return make_constant(
+                           type, shape, std::pow(2, (output_type == element::i8) ? -24 : -23)) *
+                       (max_abs32 / max_abs8);
+            }
+
+            std::shared_ptr<Node> get_bias_scale(std::shared_ptr<Node> min_input,
+                                                 std::shared_ptr<Node> max_input,
+                                                 std::shared_ptr<Node> min_filter,
+                                                 std::shared_ptr<Node> max_filter)
+            {
+                auto type = min_input->get_element_type();
+                if (type != max_input->get_element_type() ||
+                    type != min_filter->get_element_type() ||
+                    type != max_filter->get_element_type())
+                {
+                    throw ngraph_error("get_scale: min and max must have same type");
+                }
+
+                auto shape = min_input->get_shape();
+                if (shape != max_input->get_shape() || shape != min_filter->get_shape() ||
+                    shape != max_filter->get_shape())
+                {
+                    throw ngraph_error("get_scale: min and max must have same shape");
+                }
+
+                auto max_abs_input_range = max_abs(min_input, max_input);
+                auto max_abs_filter_range = max_abs(min_filter, max_filter);
+                auto range = make_constant(type, shape, 255.0 * 127.0);
```

Comment:
I would still prefer these not be "magic numbers".  This is std::numeric_limits<uint8_t>::max() and std::numeric_limits<int8_t>::min()?